app/models/fax_recipient.rb

```diff
@@ -11,6 +11,10 @@ def name
     office["name"]
   end
 
+  def email
```

Comment:
now that we have this, I wonder if the class name should no longer be `FaxRecipient` ?

Reply:
Great call! Agree 100%